src/requests/requests.cpp

```diff
@@ -113,4 +113,21 @@ bool CreateAndDownloadFile(const std::string &url, const std::string &path, int
   return true;
 }
 
+auto DownloadToStream(char const *url, std::ostream &os) -> bool {
```

Comment:
@Ignition Once users start using this feature the wrong way, it would be helpful if we added a few logs here and there, `spdlog::info` for users (non-recuring and informative)  and `spdlog::trace` for us. 
This could be added to the PR where you see it fits.

Reply:
Added SPDLOG_WARN to be consistent with the rest of requests.cpp. The CSV stuff already has exceptions. Also removed `tftp` and `telnet` because they could be problematic. probably wait for a user to request those before offering them.